### DISCUSSION TITLE
Wiped production DB by mistakes, any options?

### DISCUSSION THREAD

**Discussion by theSa1** [vercel/community#2901] [Help] [answered]
### Summary

So i kinda f'd up,

here is how it went,
i ran prisma migrate dev on production by mistake, realizing this mistake, i immediately control c'd task,
at this point i did not think this would cause whole db to be wiped, so i continued as nothing had happen,
but the server was constantly erroring, wondering why i looked to vercel storage (where i hosted my db)
i saw no tables there, i panicked and very stupidly ran prisma db push
and so now i have no db and probably no job in a few days time

is there any option? please i would really appreciate even a bit of help

### Steps to Reproduce

N/A

**Comment by Pieparker** [comment]
Hey there! Please share with us the `project_id` of the impacted project. You can find this by navigating to `https://vercel.com/[username]/[project-name]/settings/general`.

**Reply by amyegan** [reply]
@theSa1 If you can also share the date and time when the db was wiped, that will let us know how far back to go in the data history 🙏

**Comment by theSa1** [comment]
Hello,

It was on 18th of June, at around 11:30 AM IST, thank you so much for
responding, and i would be very thankful if you can recover

**Reply by theSa1** [reply]
Also the project I'd is prj_cIp9sdTSUYAwFPx4YChLR9ENAVFS

**Comment by amyegan** [comment, marked as answer]
I asked the Storage team for help checking on this, but I’m afraid I don’t have good news to share. 😞

We don’t have the history available to restore. The Postgres solution is still in beta so some features, including backups, are still in development. I’m sorry that it wasn’t ready in time to save your data 🙇
